src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Program.cs

```diff
@@ -69,11 +70,18 @@ public static async Task MainAsync(string[] args)
                     .WithOutput(Console.OpenStandardOutput())
                     .ConfigureLogging(builder => builder
                         .AddLanguageServer()
-                        .SetMinimumLevel(logLevel))
+                        .SetMinimumLevel(RazorLSPOptions.GetLogLevelForTrace(trace)))
```

Comment:
I don't know if we have power over this but I don't care for this name. `SetLogLevel` or `SetMinimumLogLevel` gives people more of an idea what it is without having to guess based on the parameters.

Reply:
Nope. This is a public api in our logging stack.

Reply:
:(